var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -169,6 +173,11 @@ def configure(self):
     def configure(self):
         configure('-fast',
                   '-no-webkit',
+                  # XXX(osx): sdk stuff; should depend on the architecture and the SDKs available.
+                  '-arch', 'x86_64',
+                  '-sdk', '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk',
```

Comment:
indeed, i only have `MacOSX10.11.sdk` available there. Maybe still query for a proper sdk path in a hope that a compiler error will be fixed at some point?